<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoulderSelectionWindow</class>
 <widget class="QMainWindow" name="ShoulderSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Planification épaule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="options_frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QGroupBox" name="display_options">
           <property name="title">
            <string>Affichage</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QCheckBox" name="displaySagital">
                <property name="text">
                 <string>Plan Sagital</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="displayAxial">
                <property name="text">
                 <string>Plan Axial</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="displayCoronal">
                <property name="text">
                 <string>Plan Coronal</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="displayFriedman">
                <property name="text">
                 <string>Axe Friedman</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="display_wireframe">
                <property name="text">
                 <string>Afficher en transparence</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="planification_axis">
           <property name="title">
            <string>Plannification</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="init_plannif_button">
                <property name="text">
                 <string>Initialiser</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="inclinaison_input">
                  <property name="minimum">
                   <double>-99.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Inclinaison (°)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Version (°)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="version_input">
                  <property name="minimum">
                   <double>-99.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                  <property name="stepType">
                   <enum>QAbstractSpinBox::DefaultStepType</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="export_button">
                <property name="text">
                 <string>Exporter le resultat</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="points_definition">
           <property name="title">
            <string>Definition de points</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="selectGlene">
                <property name="text">
                 <string>Centre Glène</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="selectTrigone">
                <property name="text">
                 <string>Trigone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="selectScapula">
                <property name="text">
                 <string>Scapula</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="selectNone">
                <property name="text">
                 <string>Fini</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="selection_screen" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>531</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="first_model_loading_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Charger un modèle STL pour commencer</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIchier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCharger"/>
    <addaction name="separator"/>
    <addaction name="actionExporter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAide"/>
   </widget>
   <addaction name="menuFIchier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Exporter</string>
   </property>
  </action>
  <action name="actionSelection_Centre_Glene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selection Centre Glene</string>
   </property>
   <property name="toolTip">
    <string>Selectionner le centre Glène</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
